full/src/test/java/apoc/mongodb/MongoDBTest.java

```diff
@@ -247,6 +247,7 @@ public void testDelete() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Can you leave TODOs for these tests please? A TODO somehwere is easily searchable, the @Ignore task alone may not give us enough information on whether we need to un-ignore these tests